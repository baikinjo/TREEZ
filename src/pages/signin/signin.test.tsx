import React from 'react'
import { shallow } from 'enzyme'
import SingInPage from './signin'

it('should render sign in page component', () => {
  expect(shallow(<SingInPage />)).toMatchSnapshot()
})
