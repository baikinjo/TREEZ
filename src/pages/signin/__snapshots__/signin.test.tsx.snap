// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render sign in page component 1`] = `ShallowWrapper {}`;
